Add note on --rpc-gas-option (#1599)
hyperledger/besu-docs: docs/public-networks/reference/cli/options.md

</Tabs>

Sets a limit on the amount of gas for transaction simulation RPC methods. 
Its value must be greater than or equal to `0`. 
The default is `0`, which indicates there is no limit. 
This cap prevents [`eth_call`](../api/index.md#eth_call) requests from using excessive resources.

:::note

> This option allows users to override the transaction's gas limit. 

Reviewer: Can we combine this sentence with, "Sets a limit on the amount of gas for transaction simulation RPC methods. " in the opening paragraph? We can then remove it from the note.

> It helps simulate transactions that exceed anticipated gas usage without going over a predefined cap, preventing DoS attacks.

Reviewer 2: ```suggestion
This can prevent the simulation of transactions with high gas usage by setting a predefined cap, preventing DoS attacks.
```
